IGN.js

```diff
@@ -1,103 +1,105 @@
 /*
-	IGN Translator - Parses IGN articles and creates Zotero-based metadata
-	Copyright (C) 2010-2011 odie5533
+	***** BEGIN LICENSE BLOCK *****
 
-	This program is free software: you can redistribute it and/or modify
-	it under the terms of the GNU General Public License as published by
+	Copyright © 2016 czar
+	http://en.wikipedia.org/wiki/User_talk:Czar
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
 	the Free Software Foundation, either version 3 of the License, or
 	(at your option) any later version.
 
-	This program is distributed in the hope that it will be useful,
+	Zotero is distributed in the hope that it will be useful,
 	but WITHOUT ANY WARRANTY; without even the implied warranty of
-	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-	GNU General Public License for more details.
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
 
-	You should have received a copy of the GNU General Public License
-	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
 */
 
-function detectWeb(doc, url) {
-	if (url.match(/articles/)) {
-		return "webpage";
-	}
-}
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
 
-function scrape(doc, url) {
-	var newItem = new Zotero.Item("webpage");
-	newItem.publicationTitle = "IGN";
-	newItem.url = doc.location.href;
-	newItem.title = doc.title.replace(/ -[^-]+IGN/, "");
-	
-	// pages
-	var pages = doc.evaluate('//div[@class="ui-page-list clear"]/ul/li[last()-1]', doc, null, XPathResult.ANY_TYPE, null);
-	if (p = pages.iterateNext())
-		newItem.pages = p.textContent;
-	
-	// date
-	var dates = doc.evaluate('//div[@class="article_pub_date"]/text()|//span[@class="article-publish-date"]/text()', doc, null, XPathResult.ANY_TYPE, null);
-	newItem.date = dates.iterateNext().textContent.replace(/^\s+|\s+$/g,'');
-	
-	//authors
-	var byline = doc.evaluate('//div[@class="article_author"]|//span[contains(@class,"article-byline")]', doc, null, XPathResult.ANY_TYPE, null);
-	var authors = byline.iterateNext().textContent.replace(/^by\s*/i, "").split(" and ");
-	for (var i=0; i<authors.length; i++) {
-		newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i], "author"));
-	}
-	
-	// attach html
-	newItem.attachments.push({title:"IGN Article Snapshot", document:doc});
-	
-	newItem.complete();
-}
+FW.Scraper({
+itemType         : 'blogPost',
+detect           : FW.Url().match(/articles\/\d{4}/),
```

Comment:
given that you only detect on ign.com/articles/, could you add /articles/ to the target (leave it in detect to be safe)